src/contrib/images.jl

```diff
@@ -0,0 +1,78 @@
+#=
+ImageTransformations
+=#
+# function warp(img::AstroImageMat, args...; kwargs...)
+#     out = warp(arraydatat(img), args...; kwargs...)
+#     return copyheaders(img, out)
+# end
+
+
+
+# Instead of using a datatype like N0f32 to interpret integers as fixed point values in [0,1],
+# we use a mappedarray to map the native data range (regardless of type) to [0,1]
+ImageCore.normedview(img::AstroImageMat{<:FixedPoint}) = img
+function ImageCore.normedview(img::AstroImageMat{T}) where T
+    imgmin, imgmax = extrema(skipmissingnan(img))
+    Δ = abs(imgmax - imgmin)
+    # Do not introduce NaNs if limits are identical
+    if Δ == 0
+        Δ = one(imgmin)
```

Comment:
This should be `oneunit` rather than `one`.

Reply:
Thanks!